### ISSUE TITLE
[BUG] Vue-formio is not work with bootstrap 5

### ISSUE THREAD

**Issue by shuhaib-aot** [formio/vue#65]
i am work with bootstrap modal 5 but Form is not working when i giving src url

**Comment by Sefriol** [comment]
I think formio does not support Bootstrap 5 yet.

**Comment by travist** [comment]
We have been working on releasing a bootstrap 5 version of our libraries starting with our formio/formio.js repo. See here for the pull request that implements this.

https://github.com/formio/formio.js/pull/4607

This has been pulled into the latest version, which is currently released as 5.0.0-rc.1. I have not tried it personally, but you may be able to get it to cooperate by installing this version alongside this library like so.

```
yarn install @formio/vue
yarn install @formio/formiojs@5.0.0-rc.1
```

Give it a try and let us know if you run into issues.

**Comment by daneformio** [comment]
Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
